index.html

```diff
@@ -1,15 +1,15 @@
-<!DOCTYPE html>
-<html lang="ja">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Summer_Contest_2022</title>
-
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <canvas id = "canvas1"></canvas>
-    <script src = "script.js"></script>
-</body>
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Summer_Contest_2022冬の未提出test＿1234567890</title>
```

Comment:
```suggestion
    <title>Summer_Contest_2022夏の未提出</title>
```
冬でなく夏ですよ
コメントをココに書く！